packages/subgraph/schema.graphql

```diff
@@ -1,33 +1,47 @@
 interface IPlugin {
-  id: ID! # PluginEntityID
+  id: ID! # Plugin Address
```

Comment:
Initially, I thought that was Plugin + Address until I realized it had no sense, so maybe lower case `address` could be clear 
``` suggestion
  id: ID! # Plugin address
```